azure-pipelines.yml

```diff
@@ -4,246 +4,241 @@
 # https://aka.ms/yaml
 
 trigger:
-- master
```

Comment:
Are these just formatting?

Reply:
Seem to be, I didn't notice this change. Prettier must have reformatted things when I added the package generation command.

Reply:
I had to rebase and in the process had to redo this change and can confirm that it's because prettier does not like what we have on master currently. I'd recommend configuring prettier on your machine to run on file save.